coffee-shop/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Please create the new folder with your name appended to the coffee-shop folder and add all your changer there.